editor/script_create_dialog.cpp

```diff
@@ -694,7 +694,10 @@ void ScriptCreateDialog::_update_dialog() {
 		parent_name->set_editable(true);
 		parent_search_button->set_disabled(false);
 		parent_browse_button->set_disabled(!can_inherit_from_file);
-		_msg_path_valid(true, TTR("Built-in script (into scene file)."));
+		_msg_path_valid(
+			true,
+			TTR("Built-in script (into scene file).\n    Note: Built-in scripts have some limitations\n    and can't be edited using an external editor.")
```

Comment:
We shouldn't have to do manual wrapping, that's pretty bad for translations.

The code wrapping on several lines also doesn't really improve readability IMO.

Reply:
I remade the implementation to use a separate autowrapped label instead. It looks better now :slightly_smiling_face:

Reply:
Can't we make the main status label autowrapped directly?

Reply:
We could, but I think having a separate label is better so the notice doesn't display in green (which looks a bit strange in this context).

Still, it would be good to make the main status labels use autowrapping for better localization support. I've tried to make them use autowrapping, but it causes the dialog to extend all the way up (making it very tall for no reason). This only happens the first time the dialog appears though…